### PR TITLE
Performance improvement

### PR DESCRIPTION
**Pull request by nhosoi**
- Eliminating unnecessary set_fact tasks in tasks/main.yml and moving
  them to the individual input and output.
- Eliminating unnecessary loop for the set_certs tasks.
- Fixing a typo and applying the set_certs changes to tests_ovirt_elasticsearch.yml
  and tests_files_elasticsearch.yml.

### CODE REVIEW

**Comment by nhosoi** [comment]
[citest bad]

File: tasks/main.yml [hunk 1]
```diff
@@ -2,65 +2,28 @@
...
-    - name: "Set rsyslog_output_remote_files for the omfwd output
-      for the remote input"
+    - name: Set rsyslog_outputs
       set_fact:
-        rsyslog_output_remote_files: "{{ logging_outputs |
-          selectattr('name', 'defined') | selectattr('type', 'defined') |
-          selectattr('type', 'match', '^remote_files$') | list }}"
+        rsyslog_outputs: "{{ logging_outputs | selectattr('name', 'defined') |
+          selectattr('type', 'defined') }} +
+          {{ _rsyslog_output_files | d([]) }}"
```
Comment (spetrosi) [inline tasks/main.yml:20]: Not a correction but a question, why some variables you introduced are prefixed with one underscore, not two underscores as usual? These are `_rsyslog_output_files`, `_input`, `_output`.

**Comment by nhosoi** [comment]
[citest bad]

File: tasks/main.yml (hunk 1 above)
Comment (nhosoi) [reply tasks/main.yml:20]: Ah, I did not think about it much... :p Thanks for pointing it out, @spetrosi. I'm going to replace them with two underscores.

Comment (nhosoi) [reply tasks/main.yml]: @spetrosi, fixed. I also replaced `_input` and `_output` with `__rsyslog_input` and `__rsyslog_output` to make them less ambiguous. Thanks, again!

Comment (spetrosi) [reply tasks/main.yml]: Thank you for doing this, that's much clearer. I would wait for the CI results now.

**Comment by nhosoi** [comment]
[citest pending]

**Comment by nhosoi** [comment]
I'm wondering why baseos CI tests are not launched...

**Review by spetrosi — APPROVED** [review]
lgtm

**Comment by nhosoi** [comment]
Thank you for your reviews, @spetrosi! Merging...

**Pull request merged**